packages/docs-app/src/examples/core-examples/menuExample.tsx

```diff
@@ -57,3 +59,12 @@ export class MenuExample extends BaseExample<{}> {
         );
     }
 }
+
+const PalantirLogo: React.SFC = () => (
+    <svg className={Classes.ICON} width="16" height="16" viewBox="0 0 18 23" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@adidahiya maybe some props should be passed into given icon element?
especially if it's an `svg` element:  `className`, `iconSize`, `color`.
or we could support a `paths` prop?

Reply:
no, I'd rather not add that complexity to `<Icon>`